vsintegration/Vsix/VisualFSharpFull/VisualFSharpFull.csproj

```diff
@@ -14,11 +14,15 @@
     <None Include="Source.extension.vsixmanifest">
       <SubType>Designer</SubType>
     </None>
-    <Content Include="..\RegisterFsharpPackage.pkgdef">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-      <Link>RegisterFsharpPackage.pkgdef</Link>
-    </Content>
+
+    <NoneSubstituteText Include="..\RegisterFsharpPackage.pkgdef" CopyToOutputDirectory="PreserveNewest">
```

Comment:
This probably won't work unless we specially handle `@(ContentSubstituteText)`, otherwise the VSSDK probably won't find the `.pkgdef` in the `@(None)` collection.

Reply:
@brettfo I believe the VSSDK picks the packagedefs out of the  output directory and since the NoneSubstituteText puts the transformed file there, all is good..

Reply:
@brettfo, Okay fixed it.